### PR TITLE
Prepare version 2.3.0 #774

### PR DESCRIPTION
**Pull request by Mrlunsuper**
### Changes proposed in this Pull Request:
Version 2.3.0 ready for release.

### CODE REVIEW

**Review by ketulsheth85 — APPROVED** [review]

File: class-wc-vendors.php [hunk 1]
```diff
@@ -7,11 +7,11 @@
  * Author URI:           https://www.wcvendors.com
  * GitHub Plugin URI:    https://github.com/wcvendors/wcvendors
  *
- * Version:              2.2.4
+ * Version:              2.3.0
  * Requires at least:    5.3.0
- * Tested up to:         5.7
+ * Tested up to:         5.7.2
```
Comment (digitalchild) [inline class-wc-vendors.php:12]: We need to test with the soon to be released WP5.8 as this is now in beta

File: class-wc-vendors.php [hunk 2]
```diff
@@ -7,11 +7,11 @@
...
  * WC requires at least: 4.0
- * WC tested up to:      5.2
+ * WC tested up to:      5.4.1
```
Comment (digitalchild) [inline class-wc-vendors.php:14]: WooCommerce 5.5.0 is scheduled to be released tomorrow. We should be doing all of our testings with the latest beta's as well as the latest final releases.

File: changelog.txt [hunk 3]
```diff
@@ -1,5 +1,18 @@
 Changelog for WC Vendors Marketplace
 
+Verison 2.3.0 - 6th July 2021
```
Comment (digitalchild) [inline changelog.txt:3]: Update this date to the 14th of July as it will take us at least till the end of tomorrow to finish testing.

**Review by digitalchild — CHANGES_REQUESTED** [review]
See my comments. You will need to update the readme.txt to ensure the tested versions are the latest betas.

**Review by digitalchild — APPROVED** [review]

**Review by bdesai1982 — APPROVED** [review]

**Pull request merged**